spec/unit/analytics/google-analytics-universal-tracker.spec.js

```diff
@@ -4,14 +4,14 @@ describe("GOVUK.GoogleAnalyticsUniversalTracker", function() {
   beforeEach(function() {
     window.ga = function() {};
     spyOn(window, 'ga');
-    universal = new GOVUK.GoogleAnalyticsUniversalTracker('id', 'cookie-domain.com');
+    universal = new GOVUK.GoogleAnalyticsUniversalTracker('id', 'cookie-domain.com', { siteSpeedSampleRate: 100 });
   });
 
   it('can load the libraries needed to run universal Google Analytics', function() {
     delete window.ga;
-    $('[src="//www.google-analytics.com/analytics.js"]').remove();
+    $('[src="https://www.google-analytics.com/analytics.js"]').remove();
```

Comment:
Are we sure current instances match this new source? If we need both maybe we should select both.